hypothesis-python/src/hypothesis/_settings.py

```diff
@@ -411,9 +413,32 @@ class Setting(object):
     deprecated_since = attr.ib()
 
 
+def _ensure_positive_int(x, name, min_value=1):
+    if not isinstance(x, integer_types):
+        note_deprecation(
+            "Passing non-integer %s=%r is deprecated" % (name, x), since="RELEASEDAY"
```

Comment:
I don't think this use of `RELEASEDAY` does what you want, because it will be textually replaced at release time, which means that future use of this code will show up as deprecated since whenever we release this patch.